src/ixdat/plotters/ec_plotter.py

```diff
@@ -12,18 +13,24 @@ def __init__(self, measurement=None):
         """Initiate the ECPlotter with its default Meausurement to plot"""
         self.measurement = measurement
 
+    @deprecate("0.1", "Use `v_name` instead.", "0.3", kwarg_name="V_str")
```

Comment:
Ha. I had never, annoyingly, consider how these would stack. Presumably the `@wraps` decorator copies the necessary callable signature over, otherwise it wouldn't work. Have you tested that it works both for more than the first argument name "j_color", because I don't test that.